docs/2022-04-07-Full-Corectness-All-Path-Reachability-Proofs.md

```diff
@@ -0,0 +1,412 @@
+Proving Full Correctness All-Path Reachability Claims
```

Comment:
Does the term "Full Correctness" mean total correctness (see [correctness](https://en.wikipedia.org/wiki/Correctness_(computer_science)#:~:text=Correspondingly%2C%20to%20prove%20a%20program's,the%20halting%20problem%20is%20undecidable.))?